backend_new/test/integration/permission-tests/permission-as-juris-admin-wrong-juris.e2e-spec.ts

```diff
@@ -0,0 +1,1086 @@
+        },
+      });
+      await request(app.getHttpServer())
+        .get(
+          `/listings/byMultiselectQuestion/${listingACreated.listingMultiselectQuestions[0].multiselectQuestionId}`,
+        )
+        .set('Cookie', cookies)
+        .expect(200);
+    });
+
+    it('should error as forbidden for delete endpoint', async () => {
+      const listingData = await listingFactory(jurisId, prisma);
+      const listing = await prisma.listings.create({
+        data: listingData,
+      });
+
+      await request(app.getHttpServer())
+        .delete(`/listings/`)
+        .send({
+          id: listing.id,
+        } as IdDTO)
+        .set('Cookie', cookies)
+        .expect(403);
+    });
+
+    it('should error as forbidden for update endpoint', async () => {
+      const listingData = await listingFactory(jurisId, prisma);
+      const listing = await prisma.listings.create({
+        data: listingData,
+      });
+
+      const val = await constructFullListingData(prisma, listing.id, jurisId);
+
+      await request(app.getHttpServer())
+        .put(`/listings/${listing.id}`)
+        .send(val)
+        .set('Cookie', cookies)
+        .expect(403);
+    });
+
+    it('should error as forbidden for create endpoint', async () => {
+      const val = await constructFullListingData(prisma, undefined, jurisId);
+
+      await request(app.getHttpServer())
+        .post('/listings')
+        .send(val)
+        .set('Cookie', cookies)
+        .expect(403);
+    });
+  });
+
+  describe('Testing application flagged set endpoints', () => {
+    it('should succeed for list endpoint', async () => {
+      await request(app.getHttpServer())
+        .get(`/applicationFlaggedSets?`)
+        .set('Cookie', cookies)
+        .expect(200);
+    });
+
+    it('should succeed for meta endpoint', async () => {
+      await request(app.getHttpServer())
+        .get(`/applicationFlaggedSets/meta?`)
+        .set('Cookie', cookies)
+        .expect(200);
+    });
+
+    it('should succeed for retrieve endpoint', async () => {
+      const listing = await createSimpleListing(prisma, jurisId);
+      const applicationA = await createSimpleApplication(prisma);
+      const applicationB = await createSimpleApplication(prisma);
+
+      const resolvedAFS = await prisma.applicationFlaggedSet.create({
+        data: {
+          rule: RuleEnum.email,
+          ruleKey: `example rule key ${listing}`,
+          showConfirmationAlert: false,
+          status: FlaggedSetStatusEnum.resolved,
+          listings: {
+            connect: {
+              id: listing,
+            },
+          },
+          applications: {
+            connect: [{ id: applicationA }, { id: applicationB }],
+          },
+        },
+      });
+
+      await request(app.getHttpServer())
+        .get(`/applicationFlaggedSets/${resolvedAFS.id}`)
+        .set('Cookie', cookies)
+        .expect(200);
+    });
+
+    it('should succeed for resolve endpoint', async () => {
```

Comment:
Feeling a little lost on why juris admin would be able to resolve afs in another jurisdiction, could you explain this?

Reply:
Oh wait, I'm not assuming that this is just a product of the [issue](https://github.com/bloom-housing/bloom/pull/3742#pullrequestreview-1763627773) I mentioned above, is the plan to handle that at a later date?

Reply:
Hey @ColinBuyck you're right that probably shouldn't be allowed. We have a ticket to investigate switching off of casbin since its ability to permission individual records is a little shoddy. 

I think that since resolving a flagged set does require knowing the applications and a jurisdictional admin cannot get a list of application ids for listings not in their jurisdictions, we are somewhat protected from a jurisdictional admin randomly resolving a flagged set not in their jurisdiction.

Lemme know if thats okay

Reply:
Sounds okay to me, thanks for the explanation!